src/katcp_fpga.py

```diff
@@ -218,6 +226,7 @@ def program(self, filename=None):
             elif filename != self.system_info['program_filename']:
                 LOGGER.error('Programming filename %s, configured programming filename %s'
                              % (filename, self.system_info['program_filename']))
+                # This doesn't seem as though it should really be an error...
```

Comment:
Well, the idea was that you could set the self.system_info['program_filename'] variable and then just say self.program(). So if you'd set it, and then passed in a different filename, that wouldn't be correct.